index.html

```diff
@@ -1981,6 +1988,52 @@ <h3 id="cryptosuiteString">The `cryptosuiteString` Datatype</h3>
           </dl>
         </section>
 
+        <section>
+          <h4 id="multibase">The `multibase` Datatype</h3>
+
+          <p>
+This specification encodes binary data into [[?MULTIBASE]]-encoded strings,
+which is useful in environments that cannot represent some binary values, such
+as ASCII-only data formats. In environments that support data types
```

Comment:
```suggestion
[[?MULTIBASE]]-encoded strings are used to encode binary data into ASCII-only format, useful in environments that cannot directly represent binary values. This specification makes use of this encoding. In environments that support data types
```

I must admit, I have difficulties parsing this sentence. 

- This specification does not really encode: it sounds as if it was defining Multibase...
- Why "some" binary values? Isn't it the case that the idea is to use ASCII characters to encode _any_ binary values?
- The "such as ASCII-only data format" reads to my non-native ears as if ASCII only data format would be one example for the "some binary values"

My editing is just an attempt to have something I understand, but that may be wrong.

Reply:
Your text is better, I'm going to merge it and then reword it if necessary.

I was waffling on the language because there is a new draft entry for base256-emoji (yes, really), which allows one to use UTF-8 strings to encode (which would be an example of a non-ASCII encoding). All that said, it'll be a while before base256-emoji makes it in (if ever, since it's benefits are slim -- still trying to figure out if it was a joke or not).